Library/Homebrew/api.rb

```diff
@@ -64,6 +64,7 @@ def self.fetch_json_api_file(endpoint, target:)
             ohai "Downloading #{url}" if $stdout.tty?
             # Disable retries here, we handle them ourselves below.
             Utils::Curl.curl_download(*args, url, to: target, retries: 0, show_error: false)
+            File.touch(target)
           end
```

Comment:
```suggestion
          end
          File.touch(target)
```

Are you sure we don't need to still touch if we skipped the download?

Reply:
That would just cause it to drift to 450 seconds after the last brew command rather than 450 seconds after the last update, no?